content/60.methods.md

```diff
@@ -57,40 +70,27 @@ By recording the evolution of information over time and assembling a resource th
-The current project was managed through GitHub [@url:https://github.com/greenelab/covid19-review] using Manubot [@doi:10.1371/journal.pcbi.1007128] to continuously generate a version of the manuscript online [@url:https://greenelab.github.io/covid19-review].
+A preliminary requirement for this undertaking was to establish Manubot as a platform accessible to researchers with limited experience working with git, as is common in biology and medicine, where version control is not typically emphasized [@doi:10.1109/SE4Science.2017.11; @doi:10.1177/2515245918754826; @doi:10.1186/1751-0473-8-7].
 Contributors were recruited by word of mouth and on Twitter, and we sought out opportunities to integrate existing efforts to train early-career researchers (ECRs).
-Few researchers in biological and medical fields are trained in version control tools such as git <!--To Do: Find some sort of reference for this, software carpentry? Pull from citations to 10.1371/journal.pcbi.1004668
--->
-
-In order to make the project accessible to individuals from a number of backgrounds, we developed resources explaining how to use GitHub's web interface to develop and edit text and interact with Manubot for individuals with no prior experience working with git or other version control platforms.
-<!--To Do: elaborate on resource development-->
-
-Interested parties were encouraged to contribute in a number of ways.
-One option was to submit articles of interest as issues in the GitHub repository.
-Articles were classified as _diagnostic_, _therapeutic_, or _other_, and a template was developed to guide the review of papers and preprints in each category.
-Following a framework often used for assessing medical literature, the review consisted of examining methods used in each relevant article, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367].
-For examples of each template, please see Appendices B-D.
-Another option was to contribute or edit text using GitHub's pull request system.
-Each pull request was reviewed and approved by at least one other author.
-Manubot also provides a functionality to create a bibliography using digital object identifiers (DOIs), website URLs, or other identifiers such as PubMed identifiers and arXiv IDs.
+We invited potential collaborators to contribute a short introduction on a GitHub issue in order to collect information about who was involved and provide an introduction to working with GitHub issues.
+Interested participants were encouraged to contribute in a number of ways.
+One option was to catalog articles of interest as issues in the GitHub repository.
+We developed a standardized set of questions for contributors to consider when evaluating an article following a framework often used for assessing medical literature.
```

Comment:
Are we including these in an appendix?  Are we even allowed to have an appendix?

Reply:
I'm assuming no appendices are allowed, so I just linked them in the results!